app/controllers/security_policy_rule_controller.rb

```diff
@@ -31,12 +31,12 @@ def display_destination_vms
   end
 
   def download_summary_pdf
-    assert_privileges('security_policy_rules_view')
+    assert_privileges('security_policy_rule_view')
```

Comment:
🤷 I don't know where this is in the UI but the tests now pass 😉

Reply:
wondering if these strings should be constants that both sides reference

Reply:
Yeah, that's a great idea.  It would be so much better if it raised a missing constant if you accidentally typo'd this. 

Note, even as a constant, someone would need to map those constants to what's in the fixture/database.  The whole thing is very messy because it's coming from the filesystem.  If these features came from methods/constants in a class, you also wouldn't have the mess in the tests where behavior is different if you are authorized to use that feature or not since the tests largely give admin/ see "everything" role because it's not fun to seed specific features you need for tests.

I do think it's redundant to test common UI logic for authorized vs. not authorized usage for each controller in the same way, but I think if we got the false positive (skipped) controller + action down to 0 for the routes_spec.rb, we could have that test the general authorized/not authorized and let each controller test only the unique parts of the not authorized example for each controller.